packages/bento-design-system/src/Card/Card.tsx

```diff
@@ -24,7 +25,8 @@ export function Card({ children, elevation, borderRadius, ...boxProps }: Props)
   return (
     <Box
       {...getRadiusPropsFromConfig(borderRadius ?? config.defaultRadius)}
-      className={cardRecipe({ elevation: elevation || "none" })}
+      className={cardRecipe({ elevation: elevation ?? config.defaultElevation })}
```

Comment:
I think we're preventing the elevation from being set to none in this way.
Before, it was possible to omit the "elevation" prop and get "none" here as a default value.
In the new implementation, there's no way to get the same behavior.
I think we should add "none" as a possible value in `CardElevation`, so that it can explicitly be passed as prop or be set as the `defaultElevation`.